docs/data/system/experimental-api/extend-sx-prop/extend-sx-prop.md

```diff
@@ -0,0 +1,24 @@
+# Extend the `sx` prop
```

Comment:
Let's remove back ticks from headlines - they make things feel kind of cluttered/messy.

```suggestion
# Extend the sx prop
```